carbon-design-system/carbon
# feat(icon-build-helpers): add support for fill=currentColor for icons

Add support for `fill="currentColor"` for icon assets.

#### Changelog

**New**

**Changed**

- Add `fill="currentColor"` as a default attributes

**Removed**

#### Testing / Reviewing

- Verify that icons render as expected in elements
- Verify that overrides for each path can still happen with `fill`
